contracts/strategies/mstable/IdleMStableStrategy.sol

```diff
@@ -102,7 +102,8 @@ contract IdleMStableStrategy is Initializable, OwnableUpgradeable, ERC20Upgradea
         require(token == address(0), "Token is already initialized");
 
         //----- // -------//
-        strategyToken = _strategyToken;
+        // this contract tokenizes the staked imUSD position
+        strategyToken = address(this); 
```

Comment:
the `_strategyToken` argument is no more needed right?

Reply:
well it's still used for `imUSD = ISavingsContractV2(_strategyToken);`